linux/keyman-config/debian/control

```diff
@@ -2,8 +2,10 @@ Source: keyman-config
 Maintainer: Daniel Glassey <wdg@debian.org>
```

Comment:
should really update the maintainer since it isn't Daniel any more

Reply:
k, we'll sort that out in the future #2628